scripts/release.ts

```diff
@@ -86,12 +147,25 @@ async function bumpVersions() {
     console.log(c.yellow('No version changes made. Exiting...'));
     process.exit(1);
   }
-  console.log(c.green('Bumped package versions.'));
+  console.log(c.green('Bumped package versions. Bumping dependencies next...'));
+
+  // Update the dependencies in our child design systems and examples:
+  updateChildDSAndExamples();
 
   // Update versions.json
   const currentVersionsByPackage = updateVersions();
+  // We don't want to update the versions associated with our doc site
+  sh('git reset ./packages/docs/package.json');
+  sh('git checkout ./packages/docs/package.json');
   sh('git add -u');
+  // Delete lingering package-lock.json if it exists:
+  try {
+    sh(`rm ${root}/package-lock.json`);
+  } catch {
+    console.log('No package-lock.json at the top level. Moving on...');
+  }
   console.log(c.green('Updated versions.json.'));
+  process.exit(0);
```

Comment:
Before merging, please remove

Reply:
woopsie! Thanks for catching.

Reply:
Done. I'm wondering if there is a linter rule that would've caught that?

Reply:
Maybe ya'll should have a linter for British lingo in the comments